docs/source/estimation.rst

```diff
@@ -76,33 +73,49 @@ Specification of Constraints
 a dictionary. The dictionary must contain the following entries:
 
-- 'loc' or 'query' but not both. This specifies to which subset of the
+- ``'loc'`` or ``'query'`` but not both. This specifies to which subset of the
   parameters the constraint applies. The value corresponding to 'loc' will be
   passed to df.loc and the value corresponding to 'query' will be passed to
   df.query so you can provide whatever is accepted by those methods.
-- 'type', which can take the following values:
-    - 'covariance': a set of parameters forms a valid (i.e. positive
+- ``'type'``, which can take the following values:
+    - ``'covariance'``: a set of parameters forms a valid (i.e. positive
       semi-definite) covariance matrix. This is not compatible with any other
       constraints on the involved parameters.
-    - 'sum': a set of parameters sums to a specified value. The last involved
+    - ``'sdcorr'``: the first part of a set of parameters are standard deviations,
+      the second part are the lower triangle (excluding the diagonal)
+      of a correlation matrix. All parameters together can be used to construct
+      a full covariance matrix but are more interpretable. This is not compatible
+      with any other type of constraints on the involved parameters.
+    - ``'sum'``: a set of parameters sums to a specified value. The last involved
       parameter can't have bounds. In this case the constraint dictionary also
       needs to contain a 'value' key.
-    - 'probability': a set of parameters is between 0 and 1 and sums to 1.
-    - 'increasing': a set of parameters is increasing. We check that the box
+    - ``'probability'``: a set of parameters is between 0 and 1 and sums to 1.
+    - ``'increasing'``: a set of parameters is increasing. We check that the box
       constraints are compatible with the order.
-    - 'equality': a set of parameters is restricted to be equal to a
+    - ``'equality'``: a set of parameters is restricted to be equal to a
       particular value. The value has to be specified in the constraints
       dictionary.
+    - ``'pairwise_equality'``: Two sets of parameters are pairwise equal. In this
+      the constraint dictionary has to contain the keys ``locs`` and ``queries``
+      instead of ``loc`` and ``query``. Both are lists of arbitrary length
+      and each element in the list hast to be a valid argument to
+      ``DataFrame.loc[]`` or ``DataFrame.query()``, respectively. Pairwise
+      equality constraints are just syntactic sugar and are converted
+      to normal equality constraints internally.
+    - ``'fixed'``: A set of parameters is fixed to some values. In this case
+      the constraints dictionary has to contain a ``'value'`` entry which can
+      be a scalar or an iterable of suitable length.
 
 
-Lower and upper bounds as well as fixed parameters are specified in
-:ref:`params_df`
+Lower and upper bounds are specified in :ref:`params_df`.
```

Comment:
Is it still ``params_df`` or ``params``?

Reply:
params is better, thanks.